packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -3,10 +3,7 @@ package com.datadog.reactnative
 import android.util.Log
 import android.content.pm.PackageInfo
 import com.datadog.android.DatadogEndpoint
-import com.datadog.android.core.configuration.BatchSize
-import com.datadog.android.core.configuration.Configuration
-import com.datadog.android.core.configuration.Credentials
-import com.datadog.android.core.configuration.UploadFrequency
+import com.datadog.android.core.configuration.*
```

Comment:
nit: We don't use star imports in the Android SDK codebase. To avoid Android Studio doing this you disable it in the Kotlin code style settings.